os/android/HwcService.h

```diff
@@ -0,0 +1,161 @@
+/*
+// Copyright (c) 2017 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+*/
+#ifndef OS_ANDROID_HWCSERVICE_H_
+#define OS_ANDROID_HWCSERVICE_H_
+
+
+#include <list>
+
+#include "Log.h"
+#include "libhwcservice/iservice.h"
+#include "libhwcservice/icontrols.h"
+#include "libhwcservice/idiagnostic.h"
+#include <binder/IInterface.h>
+#include <utils/String8.h>
+#include "Singleton.h"
+#include "utils_android.h"
+
+namespace android {
+//class Hwc;
+class DrmHwcTwo;
+using namespace hwcomposer;
+
+//using namespace intel::ufo::hwc::services;
+
+class HwcService : public BnService, public Singleton<HwcService>
```

Comment:
we dont need this singleton. Instead let DrmHwcTwo  own this and initialize it in init() call